src/open/ClientViewModel.js

```diff
@@ -105,6 +110,18 @@ export class ClientViewModel extends ViewModel {
                     activated: () => this.preferences.setClient(this._client.id, this._nativePlatform),
                 };
             });
+
+            if (!this._webPlatform && this.proposedDeepLink) {
+                actions.push({
+                    label: this.deepLinkLabel,
+                    url: this.proposedDeepLink,
+                    primary: true,
+                    activated: async () => {
+                        this._onDeepLinkClicked();
+                    },
```

Comment:
As a user I'd be furious if a site mutated my clipboard when I didn't click on an explicit Copy button (symbolic or textual) - I might be storing the only copy of a password in there before I move it into my password manager or similar, and OSes like macOS don't have a first party clipboard history yet.

Reply:
Thanks for the feedback,@t3chguy

We think this would be an edge case and in this instance are happy with the clipboard hack we have here - many other apps apply the same flow so it shouldn't be a huge shock to users... 

Let's keep our eyes open for GH issues and general feedback once live though as if this is a bigger problem than I realise we should re-evaluate this decision.

Reply:
Could you point to another one that does this please?

Reply:
Google Photos

Reply:
In the context of accepting an invite? Or in the context of making a shareable link?

I cannot reproduce in the latter

![image](https://user-images.githubusercontent.com/2403652/204816042-8a067744-35e1-41e9-b7c6-cee5fa173c59.png)

Clicking the link opens it, does not copy anything. Clicking copy does as expected

Reply:
User receives a link > Asked to open in the app > Agrees > Opens app/play store > Downloads or opens app > Asked to paste link > Pastes link > Link opens

I believe there are other apps that do this besides Google photos but I'd have to check.

Reply:
Sounds like that's a mobile-specific behaviour whereas here it'd affect Web/Desktop also where it definitely feels non-standard and alien.

Reply:
But adding a "copy" icon would alleviate your concerns?

Reply:
Indeed, copying in the web is limited to direct user interactions, but hiding it behind a non-obvious link is abuse of that IMO